bleak/backends/bluezdbus/manager.py

```diff
@@ -597,6 +597,22 @@ def get_device_name(self, device_path: str) -> str:
         """
         return self._properties[device_path][defs.DEVICE_INTERFACE]["Name"]
 
+    def is_connected(self, device_path: str) -> bool:
+        """
+        Gets the value of the "Connected" property for a device.
+
+        Args:
+            device_path: The D-Bus object path of the device.
+
+        Returns:
+            The current property value.
+        """
+        return bool(
+            device_path in self._properties
+            and defs.DEVICE_INTERFACE in self._properties[device_path]
+            and self._properties[device_path][defs.DEVICE_INTERFACE]["Connected"]
+        )
```

Comment:
perhaps a bit simpler/easier to read?

```suggestion
        try:
            return self._properties[device_path][defs.DEVICE_INTERFACE]["Connected"]
        except KeyError:
            return False
```

Reply:
I'm probably over thinking it but we don't get zero cost exception handling until python 3.11
https://github.com/python/cpython/issues/84403